src/Microsoft.AspNetCore.Blazor.Browser/Rendering/BrowserRendererEventDispatcher.cs

```diff
@@ -38,8 +38,10 @@ private static UIEventArgs ParseEventArgsJson(string eventArgsType, string event
                     return JsonUtil.Deserialize<UIKeyboardEventArgs>(eventArgsJson);
                 case "change":
                     return JsonUtil.Deserialize<UIChangeEventArgs>(eventArgsJson);
+                case "focus":
+                    return JsonUtil.Deserialize<UIFocusEventArgs>(eventArgsJson);
                 default:
-                    throw new ArgumentException($"Unsupported value '{eventArgsType}'.", nameof(eventArgsType));
+                    return JsonUtil.Deserialize<UIEventArgs>(eventArgsJson);
```

Comment:
Changing this to allow 'unknown' to flow through makes it possible to at least pipe through any event without data.

We might want to consider something in the future that makes this abitrarily extensible by mapping 'unknown' to something with a dictionary or JSON blob. You can imagine that components might use their own bubbling events for notifications - this pattern was used in WPF as well

Reply:
OR alternatively, we could allow you to do strongly typed things by subclassing UIEventArgs

Reply:
Yes, that would be an improvement. As part of fixing the non-bubbling events, I'll enable passthrough of `unknown` event types, mapping them simply to `UIEventHandler`. We can consider ways of including custom data as a future enhancement.

Just the notifications alone cover a large proportion of scenarios.

Reply:
BTW I've handled this in #722 so if that PR goes in first, you could revert this change to `default`.